ytmusicapi/mixins/browsing.py

```diff
@@ -252,7 +252,11 @@ def get_artist_albums(self, channelId: str, params: str) -> List[Dict]:
         body = {"browseId": channelId, "params": params}
         endpoint = 'browse'
         response = self._send_request(endpoint, body)
-        results = nav(response, SINGLE_COLUMN_TAB + SECTION_LIST_ITEM + GRID_ITEMS)
+        results = nav(response, SINGLE_COLUMN_TAB + SECTION_LIST_ITEM)
+        if 'gridRenderer' in results:
+            results = nav(results, GRID_ITEMS)
+        else:
+            results = nav(results, CAROUSEL_CONTENTS)
```

Comment:
```suggestion
        results = nav(results, GRID_ITEMS, True)
        if not results:
            results = nav(results, CAROUSEL_CONTENTS)
```

Reply:
I tried that out, but I'm getting a `TypeError: 'NoneType' object is not subscriptable`.

What does work is
```python
results = nav(results, GRID_ITEMS, True) or nav(results, CAROUSEL_CONTENTS)
```